src/azure-cli/azure/cli/command_modules/vm/_validators.py

```diff
@@ -1781,9 +1782,7 @@ def _validate_count(namespace):
         namespace.public_ip_address_dns_name,
         namespace.storage_account,
         namespace.storage_container_name,
-        namespace.subnet,
```

Comment:
why we remove it here?

Reply:
Because I disabled it. Now I decide to enable it.

Reply:
https://github.com/Azure/azure-cli/pull/17217